### PR TITLE
WIP: Readme draft for vesting data generation and deployment

### PR DESCRIPTION
**Pull request by elgatovital**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        },
+        ...
+    ],
+    ...
+]
+```
+
+### Deploying to staging
+
+1. Add generated files from data/ folder to repository and commit them.
```
Comment (rmeissner) [inline README.md:107]: I was under the impression that this would happen automatically in a github action

File: README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Generated data will be placed under `../data/guardians`
+
+Guardians list: `../data/guardians/guardians.json`
+
+Guardian images: `../data/guardians/images/`
+
+### Deploying to staging
+
+1. Add generated files from data/ folder to repository and commit them.
```
Comment (rmeissner) [inline README.md:27]: I was under the impression that this would happen automatically in a github action

Comment (elgatovital) [reply README.md:27]: Generating proofs for all allocations might take several hours. This is the main reason for doing this locally and uploading results to github.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+```
+
+If you want to see all possible parameters 
+```
+python vestings/exporter.py -h
+```
+
+Exporter will place generated files under `{output_directory}/{chain_id}`
+
+### Allocation file structure
```
Comment (DmitryBespalov) [inline README.md:60]: Thanks for the first draft! Would be also good to specify the CSV file structures for guardians and allocations

Comment (elgatovital) [reply README.md:60]: coming soon

**Pull request merged**